engine-core/scripts/test_docker_setup.py

```
@@ -19,8 +19,13 @@ def run_command(cmd, description="", check=True, capture_output=True):
     """명령어 실행 및 결과 반환"""
     print(f"🔧 {description}...")
     try:
+        # B602: shell=True → shell=False + 리스트 인자로 보안 강화
+        if isinstance(cmd, str):
+            cmd_list = cmd.split()
+        else:
+            cmd_list = cmd
```

Ruff (0.12.2)

Use ternary operator `cmd_list = cmd.split() if isinstance(cmd, str) else cmd` instead of `if`-`else`-block

Replace `if`-`else`-block with `cmd_list = cmd.split() if isinstance(cmd, str) else cmd`

(SIM108)